core/src/main/java/jenkins/model/item_category/Categories.java

```diff
@@ -0,0 +1,50 @@
+package jenkins.model.item_category;
+
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+import org.kohsuke.stapler.export.Flavor;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * It is a logic representation of a set of {$Category}.
+ */
+@ExportedBean
+public class Categories implements HttpResponse, Serializable {
```

Comment:
In fact, the `HttpResponses` class already has methods for generating JSON responses. Perhaps something could be added there for `@ExportedBean` types, keeping all of this in the one place.